tests/vfs_plus_stdio/tests/01-run.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2019 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+from testrunner import run
+
+
+def testfunc(child):
+    child.expect(r'SUCCESS')
```

Comment:
use expect_exact ?

Reply:
If you can tell me why. They do the same and there are no special symbols that could lead to problems.

Reply:
> If you can tell me why

We already had this discussion in another PR (I don't want to waste my time at searching it).

From the documentation of `expect_exact`:
`This call might be faster than expect() for two reasons: string searching is faster than RE matching and it is possible to limit the search to just the end of the input buffer.`

Reply:
> We already had this discussion in another PR (I don't want to waste my time at searching it).

Fine... Then I wasted now 2 min of my time to apply, add, fixup-commit, squash and push the change, and then have the build queue time of Murdock wasted for another 30min, so we can safe a few microseconds of test time, because a 7 character string is now just string compared instead of run through the Regex-parser...